scene/show-labels-on-layer-in-3d/src/main/java/com/esri/samples/show_labels_on_layer_in_3d/ShowLabelsOnLayerIn3dSample.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2021 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package com.esri.samples.show_labels_on_layer_in_3d;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.arcgisservices.LabelingPlacement;
+import com.esri.arcgisruntime.arcgisservices.LabelDefinition;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.labeling.ArcadeLabelExpression;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import com.esri.arcgisruntime.symbology.ColorUtil;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+
+public class ShowLabelsOnLayerIn3dSample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+
+      // set title, size, and add the JavaFX scene to stage
+      stage.setTitle("Show Labels on Layer in 3D");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene from an ArcGIS Online portal item
+      var portalItem = new PortalItem(
+        new Portal("https://www.arcgis.com", false), "850dfee7d30f4d9da0ebca34a533c169");
+      ArcGISScene scene = new ArcGISScene(portalItem);
```

Comment:
You could also use `var` here.

Reply:
I've avoided using `var` here since we don't name this variable explicitly `arcGISScene`: since we also have the JavaFX scene perhaps it could be confused. The guidelines for using `var` suggest only to use it when it's obvious from the local variable name what the type is, so I think sticking here with declaring the type works. 

We could consider renaming `map` and `scene` local variables to `arcGISMap` and `arcGISScene` and use `var` that way, but I think that makes it less readable.